hugo/layouts/open-source/single.html

```diff
@@ -76,38 +76,46 @@ <h2 class="text-dark col-md-10 offset-md-1">Open-source components that make mac
                 <h3 class="title-line text-dark line-yellow mt-1 mb-3">OSS Projects Highlights</h3>
             </div>
         </div>
+		
         <div class="row highlight-cards mb-4">
-            <div class="col-lg-4 mb-4 mb-lg-0">
-                <div class="card text-white orange h-100">
+		  <div class="col-lg-4 mb-4 mb-lg-0">
```

Comment:
The new changes use tabs, but the rest of the file is indented using spaced.

Reply:
I don't notice a spacing difference in the file I have? Is this a deal breaker item?